object-store/1.2/modules/ROOT/pages/object-store-to-define-a-new-os.adoc

```diff
@@ -1,46 +1,61 @@
-* You need to share state across Cluster Nodes: When you are using Mule in cluster mode, you might want to have information that is available to every node in the cluster. Global Object Stores are ideal for that.
-* You need to use information in your app's logic: The Object Store connector is not only capable of defining that store, it can also manipulate it.
+* Sharing state across components
++
+Because the Object Store is referenced by name, more than one component can use it. Suppose you have a two instances of Salesforce Connector that are configured in slightly different ways. You can use both instances with the same tokens to avoid the need to reauthenticate.
+* Sharing state across Cluster Nodes
++
+Using Mule in cluster mode, to have information that is available to every node in the cluster. 
+* Using information in your app's logic
++
+Object Store Connector is not only capable of defining the Object Store, it can also manipulate it.
 
-Global Object Stores are defined as top-level elements with a name by which other components can reference them. For example, assume that you want to create a store for access tokens:
+Global Object Stores are defined as a top-level element with a name other components can reference. The following example creates an Object Store named `tokensStore` to store access tokens:
 
 [source,xml,linenums]
 ----
 <os:object-store name="tokensStore"
-   entryTtl="1"
-   entryTtlUnit="HOURS"
-   maxEntries="100"
-   persistent="true"
-   expirationInterval="30"
-   expirationIntervalUnit="MINUTES" />
+  entryTtl="1"
+  entryTtlUnit="HOURS"
+  maxEntries="100"
+  persistent="true"
+  expirationInterval="30"
+  expirationIntervalUnit="MINUTES" />
 ----
 
-IMPORTANT: The above values apply only to an in memory object store and not to the Mule 4 cloud xref:object-store::index.adoc[Object Store v2]. In Object Store v2, unlimited keys are allowed.
+* Persistence (`persistent="true"`)
++
+Values are stored in disk and can survive a system restart. Setting `persistent` to `false` results in a transient store that stores information only in memory.
+* Expiration (`expirationInterval`)
++
+An expiration thread runs every 30 minutes and discards the elements that exceed their time to live (TTL) or their `maxEntries` limit.
+* TTL (`entryTtl`) of 1 hour
++ 
+Because access tokens are highly sensitive, you might not want to retain them for more than an hour. Every entry that exceeds the value given in `entryTtl` is automatically deleted.
+* Size limit (`maxEntries`)
++
+Anything over the limit of 100 entries are discarded when the expiration thread runs.
 
-The example defines an Object Store named `tokensStore` that has these features:
+This example is for an in-memory Object Store and doesn't provide a recommended configuration for an Object Store that stores access tokens.
```

Comment:
It is not clear what "This example" is referring to. If it refers to the one above, this should be moved before the example.